src/main/java/com/google/sps/servlets/ReviewServlet.java

```diff
@@ -33,11 +33,12 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String place_id = request.getParameter("place_id");
-    List<PlaceReviews> allLocations = getLocation(place_id);
+    List<PlaceReviews> allLocations = queryLocation(place_id);
+    PlaceReviews location = trimQuery(allLocations);
   
     // Adds the review list to a GSON/JSON object so that can be used in Javascript code    
     response.setContentType("application/json");
-    String json = new Gson().toJson(allLocations);
+    String json = new Gson().toJson(location);
```

Comment:
I'm curious to see what this object will look like once it's unrolled in javascript, hopefully it remains straightforward to access. Great idea creating a wrapper though.